linux_docker_resources/entry_point.sh

```diff
@@ -12,8 +12,6 @@ export ORIG_GID=$(echo $ORIGPASSWD | cut -f4 -d:)
 export UID=${UID:=$ORIG_UID}
 export GID=${GID:=$ORIG_GID}
 
-ARCH=`uname -i`
```

Comment:
Are you sure this is not needed anymore? Maybe it was used by `dpkg` or in one of the later scripts? I don't know if either is the case, but I was wondering if you knew specifically that it wasn't.